Packs/Pulsedive/Integrations/Pulsedive/Pulsedive.py

```diff
@@ -212,32 +201,24 @@ def domain_reputation_command(client: Client, args: Dict[str, Any], api_key) ->
             dbot_score=dbot_score
         )
 
-        domain_standard_list.append(domain_standard_context)
-        domain_data_list.append(domain_data)
+        command_results.append(CommandResults(
+            readable_output=tableToMarkdown('Domain List', domain_data),
```

Comment:
Given a single Domain, I feel that Domain List is not a good human-readable title, perhaps Domain Details?